Setup/UpgradeSchema.php

```diff
@@ -1,39 +0,0 @@
-<?php
```

Comment:
Confirming this file was deleted on purpose

Reply:
Yeah, I removed this file because it not require.